backend/reg.mli

```diff
@@ -128,4 +136,10 @@ end
 module For_testing : sig
   val get_stamp : unit -> int
   val set_state : stamp:int -> relocatable_regs:t list -> unit
+  val with_loc : t -> location -> t
+end
+
+module For_printing : sig
+  val create : name:Name.t -> typ:Cmm.machtype_component -> stamp:int ->
```

Comment:
(Maybe we should have a docstring
stating that the created register will
not be added to the list.)